config/benchmarkssuperglue/gap_bert.conf

```diff
@@ -0,0 +1,39 @@
+// This imports the defaults, which can be overridden below.
+include "../defaults.conf"  // relative path to this file
+
+project_dir = ${JIANT_PROJECT_PREFIX}
+exp_name = "bert_gap_running"  // configure this
+run_name = "run"  // default
+
+pretrain_tasks = "gap-coreference"  // empty: don't run main training phase
+target_tasks = "gap-coreference"   // train classifier only
+
+// Eval will use task-specific params.
```

Comment:
Most of the below should live in a shared config file that inherits from defaults, as final.conf does. (And many of these are already set in defaults, and should be skipped.)